playbooks/osp/provision-osp-instance.yml

```diff
@@ -29,14 +29,11 @@
       delay: 10
       timeout: 300
 
-- name: "Ensure the host is ready for Ansible"
```

Comment:
Instead of eliminating the grouping that was here before, just use a `block` to set `become: True` for the entire section.